### PR TITLE
Add Google wellformed query dataset

### PR DESCRIPTION
**Pull request by VasudevGupta7**
This pull request will add Google wellformed_query dataset. Link of dataset is https://github.com/google-research-datasets/query-wellformedness

### CODE REVIEW

File: datasets/google_wellformed_query/google_wellformed_query.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+"""
+
+_URL = "https://raw.githubusercontent.com/google-research-datasets/query-wellformedness/master/{}.tsv"
+
+
+class GoogleWellformedQuery(datasets.GeneratorBasedBuilder):
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features({"rating": datasets.Value("string"), "content": datasets.Value("string")}),
```
Comment (lhoestq) [inline datasets/google_wellformed_query/google_wellformed_query.py:44]: could you change the feature type of `rating` to float ?

you'll need to regenerate the dataset_infos.json file

**Review by lhoestq — COMMENTED** [review]
Looks really cool thanks for adding this one :) 

I left a comment.

Also can you add the tags of the dataset card ? It's the yaml part at the top of the dataset card. More info here : https://github.com/huggingface/datasets/blob/master/ADD_NEW_DATASET.md#tag-the-dataset-and-write-the-dataset-card

File: datasets/google_wellformed_query/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### Citation Information
+
+```
+@InProceedings{FaruquiDas2018,
+  title = {{Identifying Well-formed Natural Language Questions}},
+  author = {Faruqui, Manaal and Das, Dipanjan},
+  booktitle = {Proc. of EMNLP},
+  year = {2018}
+}
+```
```
Comment (lhoestq) [inline datasets/google_wellformed_query/README.md:48]: looks like you removed some paragraph titles of the dataset card

Could you add them back ?
You can leave their content as `[More Information Needed]`

**Review by lhoestq — COMMENTED** [review]
thanks for changing the type and adding the dataset card :)

**Comment by VasudevGupta7** [comment]
hope this works..

**Review by lhoestq — APPROVED** [review]
all good now thanks !

**Pull request merged**